ibis/backends/postgres/__init__.py

```diff
@@ -184,53 +194,102 @@ def list_databases(self, like=None):
             ]
         return self._filter_with_like(databases, like)
 
-    def udf(
-        self,
-        pyfunc,
-        in_types,
-        out_type,
-        schema=None,
-        replace=False,
-        name=None,
-        language="plpythonu",
-    ):
-        """Decorator that defines a PL/Python UDF in-database.
+    def create_function(self, name: str, *, schema: str | None = None) -> Callable:
+        query = f"""
+SELECT
```

Comment:
This query is a slightly modified version of what comes out of running

```
\df some_schema.*
```

in `psql`.

To see the query you need to start `psql` with `-E`